products/nexus.md

```diff
@@ -164,209 +164,239 @@ releases:
     eol: 2025-02-15
     latest: "3.59.0-01"
     latestReleaseDate: 2023-08-15
+    link: https://help.sonatype.com/en/sonatype-nexus-repository-{% assign dashedReleaseCycle = "__RELEASE_CYCLE__.0"|replace:".","-" %}{% assign dashedPatch = "__LATEST__"|replace:".","-"|split:"-"|slice:0,3|join:"-" %}{{ dashedReleaseCycle }}{% if dashedReleaseCycle != dashedPatch %}---{{ dashedPatch }}{% endif %}-release-notes.html
```

Comment:
```suggestion
    link: https://help.sonatype.com/en/sonatype-nexus-repository-3-59-0-release-notes.html
```

I would rather directly set the link than using such complicated changelog template: all of the releases using this changelog template are EOL. WDYT ?

Reply:
Agree, I will update these entries...